tst/testinstall/kernel/blister.tst

```diff
@@ -123,7 +123,7 @@ Error, ListBlist: <list> must be a small list (not the value 'fail')
 gap> ListBlist([1,2,3], fail);
 Error, ListBlist: <blist> must be a boolean list (not the value 'fail')
 gap> ListBlist([1,2,3], [true,false]);
-Error, ListBlist: <blist> must have the same length as <list> (3)
+Error, ListBlist: <blist> must have the same length as <list> (not 2 and 3)
```

Comment:
Not your fault, but rather mine (and thus not required for this PR), but I wonder if "(not 2 and 3)" could be improved... e.g. "(not 2 versus 3)"; or "(lengths are 2 and 3)" or something?